### PR TITLE
[IT] Add multiple articles in `Disambiguation`

### PR DESCRIPTION
**Pull request by G3ntl3m4n10**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Disambiguation/Combo/it.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Combo (disambiguazione)
+
+**Combo** può avere molteplici significati:
+
+- Il [moltiplicatore di punteggio attivo](/wiki/Gameplay/Combo_(score_multiplier)) visibile durante il gameplay, o il numero di oggetti colpiti di seguito senza sbagliare  
+- Una [breve catena di oggetti colpiti](/wiki/Beatmapping/Combo) che condividono lo stesso [colore](/wiki/Beatmapping/Combo_colour)
```
Comment (hhacki) [inline wiki/Disambiguation/Combo/it.md:6]:
```suggestion
- Una [breve catena di oggetti](/wiki/Beatmapping/Combo) che condividono lo stesso [colore](/wiki/Beatmapping/Combo_colour)
```

File: wiki/Disambiguation/Combo/it.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Combo (disambiguazione)
+
+**Combo** può avere molteplici significati:
+
+- Il [moltiplicatore di punteggio attivo](/wiki/Gameplay/Combo_(score_multiplier)) visibile durante il gameplay, o il numero di oggetti colpiti di seguito senza sbagliare
```
Comment (hhacki) [inline wiki/Disambiguation/Combo/it.md:5]:
```suggestion
- Il [moltiplicatore di punteggio](/wiki/Gameplay/Combo_(score_multiplier)) visibile durante il gameplay, o il numero di oggetti colpiti di seguito senza sbagliare  
```

File: wiki/Disambiguation/Mod/it.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Mod (disambiguazione)
+
+**Mod** può avere molteplici significati:
+
+- Un moderatore, come un membro [NAT](/wiki/People/Nomination_Assessment_Team) o[GMT](/wiki/People/Global_Moderation_Team)
```
Comment (hhacki) [inline wiki/Disambiguation/Mod/it.md:5]:
```suggestion
- Un moderatore, come un membro del [NAT](/wiki/People/Nomination_Assessment_Team) o del [GMT](/wiki/People/Global_Moderation_Team) 
```

File: wiki/Disambiguation/Mod/it.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Mod (disambiguazione)
+
+**Mod** può avere molteplici significati:
+
+- Un moderatore, come un membro [NAT](/wiki/People/Nomination_Assessment_Team) o[GMT](/wiki/People/Global_Moderation_Team) 
+- [Un feedback](/wiki/Modding) dato da una persona che guarda una beatmap
```
Comment (hhacki) [inline wiki/Disambiguation/Mod/it.md:6]:
```suggestion
- [Il feedback](/wiki/Modding) dato da una persona che revisiona una beatmap
```

**Review by hhacki — CHANGES_REQUESTED** [review]

File: wiki/Disambiguation/NM/it.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# NM (disambiguazione)
+
+**NM** può avere molteplici significati:
+
+- [Modalità normale](/wiki/Modding/Normal_mod)
```
Comment (hhacki) [inline wiki/Disambiguation/NM/it.md:5]:
```suggestion
- [Mod normale](/wiki/Modding/Normal_mod)
```

File: wiki/Disambiguation/NM/it.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+- La [ddifficoltà](/wiki/Beatmap/Difficulty) di una beatmap:
```
Comment (hhacki) [inline wiki/Disambiguation/NM/it.md:6]:
```suggestion
- La [difficoltà](/wiki/Beatmap/Difficulty) di una beatmap:
```

**Review by hhacki — CHANGES_REQUESTED** [review]

File: wiki/Disambiguation/NC/it.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+# NC (disambiguazione)
+
+**NC** può avere molteplici significati:
+
+- [La modalità Nightcore ](/wiki/Gameplay/Game_modifier/Nightcore)
```
Comment (hhacki) [inline wiki/Disambiguation/NC/it.md:5]:
```suggestion
- [La mod Nightcore ](/wiki/Gameplay/Game_modifier/Nightcore)
```

File: wiki/Disambiguation/NC/it.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+# NC (disambiguazione)
+
+**NC** può avere molteplici significati:
+
+- [La mod Nightcore ](/wiki/Gameplay/Game_modifier/Nightcore)
```
Comment (hhacki) [inline wiki/Disambiguation/NC/it.md:5]:
```suggestion
- [La mod Nightcore](/wiki/Gameplay/Game_modifier/Nightcore)
```

File: wiki/Disambiguation/NM/it.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+# NM (disambiguazione)
+
+**NM** può avere molteplici significati:
+
+- [Mod normale](/wiki/Modding/Normal_mod)
+- La [difficoltà](/wiki/Beatmap/Difficulty) di una beatmap:
+  - ![](/wiki/shared/diff/normal-m.png?20211215) NM, per osu!mania.
+  - ![](/wiki/shared/diff/normal-o.png?20211215) Normal, per osu!.
+  - ![](/wiki/shared/diff/normal-t.png?20211215) Muzukashii, per osu!taiko.
+  - ![](/wiki/shared/diff/normal-c.png?20211215) Salad, per osu!catch.
```
Comment (hhacki) [inline wiki/Disambiguation/NM/it.md:10]:
```suggestion
  - ![](/wiki/shared/diff/normal-c.png?20211215) Salad, per osu!catch.
  
```

**Review by hhacki — CHANGES_REQUESTED** [review]
fix remark

**Review by hhacki — CHANGES_REQUESTED** [review]

**Review by hhacki — APPROVED** [review]
lgtm

**Review by TicClick — APPROVED** [review]

**Pull request merged**